extensions/amp-ad-exit/0.1/config.js

```diff
@@ -131,8 +134,16 @@ function assertTarget(name, target, config) {
       user().assert(
           pattern.test(variable), '\'%s\' must match the pattern \'%s\'',
           variable, pattern);
-      const vendor = variable.substr(1);
-      user().assert(ANALYTICS_CONFIG[vendor], 'Unknown vendor: ' + vendor);
+      const vendor = config.targets.variableFrom3pAnalytics.vars[variable] &&
+          config.targets.variableFrom3pAnalytics.vars[variable]
+          .vendorAnalyticsSource;
```

Comment:
^^ Please give extra attention to this statement - want to be sure it's correct.